website/docusaurus.config.js

```diff
@@ -430,6 +430,19 @@ const config = {
             docId: 'introduction',
             label: 'Docs',
           },
+          {
+            type: 'doc',
+            position: 'left',
+            docId: 'playground',
+            label: 'Playground',
+            activeBaseRegex: `/docs/playground`,
+          },
+          {
+            type: 'doc',
+            position: 'left',
+            docId: 'playground',
+            label: 'Playground',
+          },
```

Comment:
As you can see, the 2nd Playground link is active matched no matter what doc page we're at, whereas the first one is only matched when we are at docs/playground.

<img width="336" alt="CleanShot 2022-09-10 at 22 41 32@2x" src="https://user-images.githubusercontent.com/51714798/189501046-8d9999ff-8251-47b8-9778-0923a56427c7.png">

<img width="320" alt="CleanShot 2022-09-10 at 22 41 42@2x" src="https://user-images.githubusercontent.com/51714798/189501048-b45bf020-cef6-4caa-80b3-c76e32abbfec.png">